### ISSUE TITLE
 in deeplabv3plus ade20k training Is layer wise decay used?

### ISSUE THREAD

**Issue by KimtaeHo3948** [open-mmlab/mmsegmentation#1920]
in deeplabv3plus ade20k training Is layer wise decay used?
in mmsegmentation training log (https://download.openmmlab.com/mmsegmentation/v0.5/deeplabv3plus/deeplabv3plus_r101-d8_512x512_80k_ade20k/deeplabv3plus_r101-d8_512x512_80k_ade20k_20200615_014139.log.json),
 learning rate starts with 0.01 
this learning rate 0.01 applied for every layer? 
Did you give the backbone a weaker lr?

**Comment by xiexinch** [comment]
Hi @KimtaeHo3948,
The learning rate is applied for every layer since in `DeepLabv3+` we do not have special settings for certain layers.
https://github.com/open-mmlab/mmcv/blob/595c2ebeb93c873ab2fc4af67a5b81640b7bc314/mmcv/runner/optimizer/default_constructor.py#L250

**Comment by KimtaeHo3948** [comment]
thank you xiexinch.!
